### ISSUE TITLE
💅 Plugin output not being reported with summary reporter

### ISSUE THREAD

**Issue by Zer0PointModule** [biomejs/biome#7580]
### Environment information

```bash
Plugin output is not being reported with the summary reporter:

example execution:

npx biome lint --reporter=summary


It will say there are errors but you don't get to see which ones.
```

### Rule name

plugin

### Playground link

https://biomejs.dev/playground/?files.main.tsx=YwBvAG4AcwBvAGwAZQAuAGwAbwBnACgAIgBoAGUAbABsAG8AIAB3AG8AcgBsAGQAIgApADsACgAKAHoALgBhAHIAcgBhAHkAKAB6AC4AcwB0AHIAaQBuAGcAKAApACkA&files.plugin.grit=YAAkAGYAbgAoACQAYQByAGcAcwApAGAAIAB3AGgAZQByAGUAIAB7AAoAIAAgACAAIAAkAGYAbgAgADwAOgAgAGAAegAuAGEAcgByAGEAeQBgACwACgAgACAAIAAgAHIAZQBnAGkAcwB0AGUAcgBfAGQAaQBhAGcAbgBvAHMAdABpAGMAKAAKACAAIAAgACAAIAAgACAAIABzAHAAYQBuACAAPQAgACQAZgBuACwACgAgACAAIAAgACAAIAAgACAAbQBlAHMAcwBhAGcAZQAgAD0AIAAiAFUAcwBlACAAPAB0AHkAcABlAD4ALgBhAHIAcgBhAHkAKAApACAAaQBuAHMAdABlAGEAZAAgAG8AZgAgAHoALgBhAHIAcgBhAHkAKAA8AHQAeQBwAGUAPgApACIACgAgACAAIAAgACkACgB9AA%3D%3D&files.biome.json=ewAKAAkAIgAkAHMAYwBoAGUAbQBhACIAOgAgACIAaAB0AHQAcABzADoALwAvAGIAaQBvAG0AZQBqAHMALgBkAGUAdgAvAHMAYwBoAGUAbQBhAHMALwAyAC4AMgAuADQALwBzAGMAaABlAG0AYQAuAGoAcwBvAG4AIgAsAAoACQAiAHYAYwBzACIAOgAgAHsACgAJAAkAIgBlAG4AYQBiAGwAZQBkACIAOgAgAGYAYQBsAHMAZQAsAAoACQAJACIAYwBsAGkAZQBuAHQASwBpAG4AZAAiADoAIAAiAGcAaQB0ACIALAAKAAkACQAiAHUAcwBlAEkAZwBuAG8AcgBlAEYAaQBsAGUAIgA6ACAAZgBhAGwAcwBlAAoACQB9ACwACgAJACIAZgBpAGwAZQBzACIAOgAgAHsACgAJAAkAIgBpAGcAbgBvAHIAZQBVAG4AawBuAG8AdwBuACIAOgAgAGYAYQBsAHMAZQAKAAkAfQAsAAoACQAiAGYAbwByAG0AYQB0AHQAZQByACIAOgAgAHsACgAJAAkAIgBlAG4AYQBiAGwAZQBkACIAOgAgAHQAcgB1AGUALAAKAAkACQAiAGkAbgBkAGUAbgB0AFMAdAB5AGwAZQAiADoAIAAiAHQAYQBiACIACgAJAH0ALAAKAAkAIgBsAGkAbgB0AGUAcgAiADoAIAB7AAoACQAJACIAZQBuAGEAYgBsAGUAZAAiADoAIAB0AHIAdQBlACwACgAJAAkAIgByAHUAbABlAHMAIgA6ACAAewAKAAkACQAJACIAcgBlAGMAbwBtAG0AZQBuAGQAZQBkACIAOgAgAHQAcgB1AGUACgAJAAkAfQAKAAkAfQAsAAoACQAiAGoAYQB2AGEAcwBjAHIAaQBwAHQAIgA6ACAAewAKAAkACQAiAGYAbwByAG0AYQB0AHQAZQByACIAOgAgAHsACgAJAAkACQAiAHEAdQBvAHQAZQBTAHQAeQBsAGUAIgA6ACAAIgBkAG8AdQBiAGwAZQAiAAoACQAJAH0ACgAJAH0ALAAKAAkAIgBhAHMAcwBpAHMAdAAiADoAIAB7AAoACQAJACIAZQBuAGEAYgBsAGUAZAAiADoAIAB0AHIAdQBlACwACgAJAAkAIgBhAGMAdABpAG8AbgBzACIAOgAgAHsACgAJAAkACQAiAHMAbwB1AHIAYwBlACIAOgAgAHsACgAJAAkACQAJACIAbwByAGcAYQBuAGkAegBlAEkAbQBwAG8AcgB0AHMAIgA6ACAAIgBvAG4AIgAKAAkACQAJAH0ACgAJAAkAfQAKAAkAfQAsAAoAIAAgACIAcABsAHUAZwBpAG4AcwAiADoAIABbAAoAIAAgACAAIAAiAC4ALwBwAGwAdQBnAGkAbgAuAGcAcgBpAHQAIgAKACAAIABdAAoAfQAKAA%3D%3D

### Expected result

I would expect to get the standard summary report that tells me which files have problems and which types of linting issues have been spotted.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by tidefield** [comment]
I might help with this. Feel free to assign to me.
